ftxwebsocket/websocket_manager.py

```diff
@@ -58,11 +59,11 @@ def wrapped_f(ws, *args, **kwargs):
 
     def _run_websocket(self, ws):
         try:
-            ws.run_forever()
+            ws.run_forever(ping_timeout=2, ping_interval=5)
         except Exception as e:
             raise Exception(f'Unexpected error while running websocket: {e}')
         finally:
-            self._reconnect(ws)
```

Comment:
Empirically, reconnection logic appears to hang when connection is lost (e.g. when internet access is disabled on the host machine). Accordingly, I've opted to flip a boolean flag on the client in this case that consumers can check for; if `lost_connection == True`, the consumer can create a new client without incurring much additional overhead relative to preexisting logic.